error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/TimeRules.java

```diff
@@ -64,6 +66,65 @@ ZoneOffset after() {
     }
   }
 
+  static final class LocalDateOfInstant {
+    @BeforeTemplate
+    LocalDate before(Instant instant, ZoneId zoneId) {
+      return Refaster.anyOf(
+          instant.atZone(zoneId).toLocalDate(),
+          instant.atZone(zoneId).toOffsetDateTime().toLocalDate(),
```

Comment:
We can have a separate `instant.atZone(zoneId).toOffsetDateTime()` -> `OffsetDateTime.ofInstant(instant, zoneId)` rule :)